omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -180,79 +206,46 @@
         "url": "reportingui/runReport.page?reportDefinition=167cf667-071d-488b-b159-d5f391774090",
         "order": 15
       },
-      {
-        "id": "ANC Register",
-        "extensionPointId": "org.openmrs.module.ugandaemr.reports.registers",
+	  {
+        "id": "HMIS 105 - Section 1",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.monthly",
         "type": "link",
-        "label": "ANC Register",
-        "url": "reportingui/runReport.page?reportDefinition=f717a684-3c78-11e7-adec-507b9dc4c741",
+        "label": "HMIS 105 - 1 - OPD",
+        "url": "reportingui/runReport.page?reportDefinition=85b9a42b-d16b-4b3c-992f-adbb806f0b00",
```

Comment:
@ssmusoke here is where its changing

Reply:
@slubwama Do not only look at the changes, since the actual ordering of the reports has changed to make sense to the end user also to ensure that the weights are streamlined. Read the whole file

Reply:
I am actually reading the whole file. But cautiously I find changes that are not related to refactoring. I Just asked why it changing. you may be having reason why it changing.
I have seen all the changes and it makes sense but only this is what I found with need to question.